include/tf-psa-crypto/private/crypto_adjust_config_derived.h

```diff
@@ -1,5 +1,5 @@
 /**
- * \file psa/crypto_adjust_config_derived.h
+ * \file tf-psa-crypto/private/crypto_adjust_config_derived.h
```

Comment:
There is a reference to "psa/crypto_adjust_config_derived.h" to update in `entropy.c`.